packages/location-field/src/styled.tsx

```diff
@@ -74,6 +76,7 @@ export const Dropdown = ({
       <DropdownButton aria-label={dropdownButtonAriaLabel} onClick={onToggle}>
         {title}
       </DropdownButton>
+      {input}
       {open && (
         <MenuItemList uniqueId={listBoxIdentifier}>{children}</MenuItemList>
```

Comment:
Internationalize the `arial-label` prop, for English change it to ~"Search Results"~ "Suggested locations".

Reply:
While we are at this component, rename `uniqueId` to `id` and don't pass props in its styled-component definition.

Reply:
We can't use `id` because it clashes with internal React props (id being an HTML reserved keyword).

Reply:
We use an `id` prop with the map, overlays, i18n etc.

Reply:
Oh good call... Totally slept through this one